desktop/lib/settings/settings-file.js

```diff
@@ -28,7 +29,12 @@ module.exports = {
 		const settingsFile = getSettingsFile();
 
 		if ( fs.existsSync( settingsFile ) ) {
-			return JSON.parse( fs.readFileSync( settingsFile ) );
+			try {
+				return JSON.parse( fs.readFileSync( settingsFile ) );
+			}
+			catch (e) {
+				debug( 'Error reading settings file' );
```

Comment:
Should we do something else here? Because we'll fall through and the app will think it's a first run (even though it isn't) and try to create a the settings file which could be problematic.